index.html

```diff
@@ -120,32 +85,54 @@
   </p>
   <p>The original Japanese version is <a href="ruby-rules-ja.pdf">available in PDF format</a>.</p>
 </section>
-	
-<section>
-<h2 id="matters-considered-by-the-simple-placement-rules">Matters considered by the simple placement rules</h2>
 
 
+
+
+<section>
+<h2 id="document_purpose">Purpose of this document</h2>
+
 <p>Ruby is the name given to the small annotations
 in Japanese content that are rendered alongside base text,
 usually to provide a pronunciation guide,
 but sometimes to provide other information.
 (See the article <a href="https://www.w3.org/International/questions/qa-ruby.en">“What is ruby”</a>
 by the internationalization Working Group
-for more information.)
-
-    <aside class=note title="Ruby as notes" id="n20200529001">
-      Notes are sometimes placed between lines
-      similarly to how ruby is laid out
-      (inter-linear notes).
-      The inter-linear notes is not covered in this document.
-      Putting a line break within ruby string is not allowed for mono-ruby. 
-      In addition, putting a line break within ruby base text is not allowed 
-      for group-ruby. 
-      In contrast, interlinear notes may have a large number of characters, 
-      putting line breaks within both the base text and the note itself
-      need to be allowed. It leads the processing of the inter-linear notes
-      be different from ruby.
+for more information.)</p>
+
+  <p>As a guide for implementers of rendering engines, this document describes a simple method of ruby composition 
+  that can be used for Japanese layout realized with technologies like CSS, SVG, and XML-FO. 
+  Unlike 
+  <a href="https://www.w3.org/TR/jlreq/"><abbr title="Requirements for Japanese Text Layout">JLReq</abbr></a>
+  [[JLREQ]], in this document only one layout method is presented for each alternative, taking into consideration best practices and important points related to Japanese layout. The points taken into consideration are described in [[[#matters-considered-by-the-simple-placement-rules]]], In addition, this document provides recommendations for double-sided ruby, where two distinct runs of ruby text are attached to the same ruby base (this is not described in  [[JLREQ]].</p>
```

Comment:
```suggestion
  [[JLREQ]], in this document only one layout method is presented for each alternative, taking into consideration best practices and important points related to Japanese layout. The points taken into consideration are described in [[[#matters-considered-by-the-simple-placement-rules]]], In addition, this document provides recommendations for double-sided ruby, where two distinct runs of ruby text are attached to the same ruby base (this is not described in  [[JLREQ]]).</p>
```

Reply:
also, comma after link to #matters-... should be period? (in the middle of this line)

Reply:
fixed